lib/errors/IdentityNotFoundError.js

```diff
@@ -2,7 +2,7 @@ const ConsensusError = require('./ConsensusError');
 
 class IdentityNotFoundError extends ConsensusError {
   /**
-   * @param {EncodedBuffer} identityId
+   * @param {Buffer} identityId
```

Comment:
Not Identifier?

Reply:
In some cases, it's a Buffer when we validate raw data. `Identifier` extends `Buffer`, so I think it's ok.